### ISSUE TITLE
ESLint couldn't find the plugin "@trilon/eslint-plugin".

### ISSUE THREAD

**Issue by Mnigos** [TrilonIO/eslint-plugin#30]
```
ESLint: 8.56.0

ESLint couldn't find the plugin "@trilon/eslint-plugin".

(The package "@trilon/eslint-plugin" was not found when loaded as a Node module from the directory "/Users/igormakowski/Documents/repositories/rigtch/rigtch-music-api".)

It's likely that the plugin isn't installed correctly. Try reinstalling by running the following:

    npm install @trilon/eslint-plugin@latest --save-dev

The plugin "@trilon/eslint-plugin" was referenced from the config file in ".eslintrc.cjs".

If you still can't figure out the problem, please stop by https://eslint.org/chat/help to chat with the team.
```

`.eslintrc.cjs`
```js
{
  plugins: [
    '@typescript-eslint',
    '@trilon/eslint-plugin',
    'import',
    'nestjs',
    'eslint-plugin-import-helpers',
    'prettier',
  ],
  extends: [
    'plugin:@typescript-eslint/strict-type-checked',
    'plugin:@typescript-eslint/stylistic-type-checked',
    'plugin:@trilon/recommended',
    'plugin:sonarjs/recommended',
    'plugin:unicorn/recommended',
    'plugin:nestjs/recommended',
    'plugin:eslint-comments/recommended',
    'prettier',
  ],
}
  ```
  
`package.json`
```json
{
  "devDependencies": {
    "@trilon/eslint-plugin": "^0.2.0",
  }
}
```

**Comment by rickdgeerling** [comment]
Thanks! You caught an error in our CI ❤️

**Comment by Mnigos** [comment]
> Thanks! You caught an error in our CI ❤️

No problem! I find your eslint plugin very useful and would really like to use it.

**Comment by rickdgeerling** [comment]
I just published v0.2.1 it's still early days for this plugin so if you encounter any more issues, please let us know and we'll fix it
